File: src/C#/BehavioralPatterns/Command/FastFoodRestaurant/Commands/FastFoodOrder.cs
using FastFoodRestaurant.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace FastFoodRestaurant.Commands
{
    public class FastFoodOrder
    {
        public List<MenuItem> FastFoodOrderMenuItems { get; private set; } = new List<MenuItem>();

        public void ExecuteCommand(OrderCommand orderCommand, MenuItem menuItem)
        {
            orderCommand.Execute(FastFoodOrderMenuItems, menuItem);
        }

        public void ShowCurrentItems()
        {
            foreach (var fastFoodOrderMenuItem in FastFoodOrderMenuItems)
            {
                fastFoodOrderMenuItem.Display();
            }

            Console.WriteLine(new string('-', 15));
        }
    }
}

File: src/JavaScript/BehavioralPatterns/Command/FastFoodRestaurant/commands/fastFoodOrder.js

const MenuItem = require('../models/menuItem');

class FastFoodOrder {
    constructor() {
        this.fastFoodOrderMenuItems = [];
    }

    executeCommand(orderCommand, menuItem) {
        orderCommand.execute(this.fastFoodOrderMenuItems, menuItem);
    }

    showCurrentItems() {
        this.fastFoodOrderMenuItems.forEach(fastFoodOrderMenuItem => {
            fastFoodOrderMenuItem.display();
        });

        console.log('-'.repeat(15));
    }
}

module.exports = FastFoodOrder;